lib/fluent/plugin/filter_process_ucs_syslog.rb

```diff
@@ -34,6 +36,11 @@ class ProcessUcsSyslog < Filter
 
     def configure(conf)
       super
+      if @domain.nil?
+        log.info("Domain not specified, assuming canary environment")
```

Comment:
Local accounts will eventually be added for all environments right? not just canary?

Reply:
yeah.

Reply:
The concept of a canary is very specific to some environments.

Here, the code needs to support authentication to UCS using both methods, domain-joined and not.